src/perturbopy/postproc/calc_modes/spectral_cumulant.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import h5py as h5
+import matplotlib.pyplot as plt
+from perturbopy.postproc.calc_modes.calc_mode import CalcMode
+
+
+class SpectralCumulant(CalcMode):
+    """
+    A class to handle and analyze spectral functions computed using the cumulant expansion approach.
+
+    Attributes:
```

Comment:
Please follow the numpy docsrting format for Attributes, Parameters, etc.

Otherwise the automatic docs generator will not work

e.g.
```
class Bands(CalcMode):
    """
    Class representation of a Perturbo bands calculation.

    Parameters
    ----------
    pert_dict : dict
    Dictionary containing the inputs and outputs from the bands calculation.

    Attributes
    ----------
    kpt : RecipPtDB
       Database for the k-points used in the bands calculation.
    bands : UnitsDict
```